python/paddle/fluid/layers/nn.py

```diff
@@ -1472,6 +1472,7 @@ def bpr_loss(input, label, name=None):
 
     helper = LayerHelper('bpr_loss', **locals())
     out = helper.create_variable_for_type_inference(dtype=input.dtype)
+    label.stop_gradient = True
```

Comment:
**We shouldn't secretly change the user's configuration.**

Reply:
This is a hot fix

Reply:
**This is not hot fix. It's introducing new bugs.**

Reply:
So... what's the best way to solve the problem?